app/assets/javascripts/routes/reviews_index.js

```diff
@@ -0,0 +1,5 @@
+App.ReviewsIndexRoute = App.AuthenticatedRoute.extend({
+  model: function() {
+    return this.modelFor('course').get('reviews');
```

Comment:
Would this end up including half-written reviews? Because we're using createRecord in the ReviewsNewRoute, so if we navigated back to the index page, would we whatever we were working on.